File: FleemanWithNET/DTO/CityDTO.cs
namespace fleeman_with_dot_net.DTO
{
    public class CityDTO
    {
        public string City_Name { get; set; }
        public string State_Name { get; set; }

        public CityDTO(string city_Name, string state_Name)
        {
            City_Name = city_Name;
            State_Name = state_Name;
        }
    }

}

File: FleemanWithJava/src/main/java/com/example/dto/CityDTO.java
package com.example.dto;

import com.fasterxml.jackson.annotation.JsonIgnore;

public class CityDTO {
    private String city_Name;
    private String state_Name;

    // constructor
    public CityDTO(String city_Name, String state_Name) {
        this.city_Name = city_Name;
        this.state_Name = state_Name;
    }

    
    public String getCity_Name() {
        return city_Name;
    }

    
    public String getState_Name() {
        return state_Name;
    }
}
